### PR TITLE
[Delivers #149494553] Front End - Upload Image

### PR DESCRIPTION
**Pull request by jonajonlee**
pictureuploadsample

### CODE REVIEW

File: apps/web-client/middleware/upload.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+module.exports = (req, res) => {
```
Comment (Harrison-M) [inline apps/web-client/middleware/upload.js]: Needs license comment

File: apps/web-client/app.js [hunk 2]
```diff
@@ -34,13 +35,16 @@ app.use(express.static(path.join(__dirname, 'public')));
...
     s3Bucket: process.env.S3_BUCKET,
   };
   next();
 });
 
 // Routes
-app.use('/', require('./middleware/homepage'));
+app.get('/', require('./middleware/homepage'));
+
+app.post('/photo', require('./middleware/upload'));
```
Comment (Harrison-M) [inline apps/web-client/app.js]: this isn't something that needs to change, but I usually don't think of end route code as "middleware"? I've always stored it in a folder called "routes" or something like that. Not sure it makes much of a difference, though, and it's technically correct.

**Review by Harrison-M — CHANGES_REQUESTED** [review]
One comment, one small change.

File: apps/web-client/app.js (hunk 2 above)
Comment (jonajonlee) [reply apps/web-client/app.js]: Ah you're right. Even the scaffolding puts them like that. In my old place, our middleware is where we put our `(req, res, next) => {...}` handlers. Just confused it. But it's not really applicable here.

In short, I'm changing these to routes

File: apps/web-client/public/javascripts/fileselect.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+$(function() {
```
Comment (Harrison-M) [inline apps/web-client/public/javascripts/fileselect.js:14]: License comment

**Review by Harrison-M — CHANGES_REQUESTED** [review]
Just a licensing thing

**Review by sreimer15 — APPROVED** [review]

**Review by Harrison-M — APPROVED** [review]

**Pull request merged**